frontend/packages/client/src/components/modals/Modal.js

```diff
@@ -10,7 +10,7 @@ export default function Modal({ children }) {
 Modal.Header = ({ title, onClose }) => {
   return (
     <div className="c-modal-header spacing-header">
-      <p className="c-modal-title">{title}</p>
+      <p className="c-modal-title is-capitalized">{title}</p>
```

Comment:
can we have this `is-capitalized` as a prop?, we have other paces using without allcaps

Reply:
@germanurrus this isn't all caps, this is sentence case (Titles Like This) which I believe should be universal throughout the app. Where would we not want this style?

Reply:
you are right, this should use the class `is-capitalized`, thanks for the clarification